fastapi-sqlalchemy/api/schema.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+import strawberry
+from strawberry.extensions import Extension
+
+from main.models import get_movies
+from main.database import SessionLocal
+
+from .definitions.movie import Movie
+
+
+class SQLAlchemySession(Extension):
+    def on_request_start(self):
+        self.execution_context.context["db"] = SessionLocal()
+
+    def on_request_end(self):
+        self.execution_context.context["db"].close()
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def ping(self) -> str:
+        return "pong"
```

Comment:
This isn't used anywhere and isn't relevant to the example at hand. Should it be removed?

Reply:
Yep will do. I usually add it at the beginning of a project because types need at least one field.